tests/driver_at24cxxx/main.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    AT24CXXX test application
+ *
+ * Tested with bluepill and AT24C256
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "xtimer.h"
+#include "board.h"
+
+#include "at24cxxx.h"
+
+#define I2C_DEVICE           I2C_DEV(0)
+
+#define WRITE_BYTE_POSITION  (12U)
+#define WRITE_BYTE_CHARACTER 'A'
+
+#define WRITE_POSITION       (444U)
+#define WRITE_CHARACTERS     { 'B', 'E', 'E', 'R', '4', 'F', 'R', 'E', 'E', \
+                               '\0' }
+
+#define SET_POSITION         (900U)
+#define SET_CHARACTER        'G'
+#define SET_LEN              (20U)
+
+int main(void)
+{
+    puts("Starting tests for module at24cxxx");
+    xtimer_init();
+
+    at24cxxx_params_t params = {
```

Comment:
You don't need to define one here but could just reuse the default one from `at24cxxx_params.h` directly in the init call:

```c
#include "at24cxxx.h"
#include "at24cxxx_params.h"

[...]

check = at24cxxx_init(&at24cxxx_dev, &at24cxxx_params[0]);
```

Reply:
Ok